### ISSUE TITLE
New feauture: Image Search

### ISSUE THREAD

**Issue by Farbmond** [benbasson/imagetoolbar#3]
Hi! Thanks for the great addon!

Is there an option to add a button for Image Reverse Search (Google, Bing, TinEye, Karma Decay and IqDB)?

I know, there is an addon like ImageLookup, but this addon has only Google and TinEye and I think the search-option would be better placed in ImageToolbar.

Farbmond

**Comment by Lemons224** [comment]
Yes! That would be awesome! I already have an addon that does this via the right-click menu, but I'd love to have that feature consolidated into Image Toolbar.
